tests/aws/services/route53resolver/test_route53resolver.py

```diff
@@ -85,6 +86,57 @@ def setup_resources(self, aws_client, cleanups):
 
         yield vpc, subnet1, subnet2, security_group
 
+    @pytest.fixture(scope="function")
+    def route53resolver_create_firewall_rules(self, aws_client, cleanups):
```

Comment:
We can just have a fixture for `create_firewall_rule` and cleanups since it was duplicated a lot of times and call it from the test itself for different params. This will be helpful in re-using this fixture even outside of this test 🙂 

Also we can remove adding a scope as it falls to this default value 😉 and just a nit: we can remove the `route53resolver` prefix in the name of the fixture. 

```
def create_firewall_rule(self, aws_client, ..):
    # create logic
    # yield the response
    # cleanup
```